### PR TITLE
feat(ci): read  package version

### PR DESCRIPTION
**Pull request by molekilla**

### CODE REVIEW

**Review by tomicvladan — APPROVED** [review]

File: scripts/release-image.sh [hunk 1]
```diff
@@ -3,15 +3,14 @@ ROOT_PATH=$(dirname "$0")
...
 CONTRACTS_IMAGE_PREFIX="fairdatasociety"
 CONTRACTS_IMAGE_URL="$CONTRACTS_IMAGE_PREFIX/$CONTRACTS_IMAGE_NAME:$BLOCKCHAIN_VERSION"
-LATEST_IMAGE_URL="$CONTRACTS_IMAGE_PREFIX/$CONTRACTS_IMAGE_NAME:latest"
 
 docker push "$CONTRACTS_IMAGE_URL"
-
-# tag image for latest
-docker tag $CONTRACTS_IMAGE_URL $LATEST_IMAGE_URL
-
-docker push "$LATEST_IMAGE_URL"
```
Comment (nugaon) [inline scripts/release-image.sh:17]: why did you remove these lines?

Comment (molekilla) [reply scripts/release-image.sh:17]: ouch

Comment (molekilla) [reply scripts/release-image.sh]: i had used a master pull before pulling latest. it should be fixed by now.

**Review by nugaon — APPROVED** [review]
